.helm/assure-hmrc-data/values/production.yaml

```diff
@@ -0,0 +1,43 @@
+# Default values for assure-hmrc-data.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+#
+# DO NOT STORE SECRETS IN HERE!
+#
+
+replicaCount: 2
+
+# image keys that need values provided by `helm install/upgrade...`
+image:
+  repository: null
+  tag: null
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  className: default
+  annotations:
+    external-dns.alpha.kubernetes.io/set-identifier: "assure-hmrc-data-laa-assure-hmrc-data-production-green"
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
+  hosts:
+    - laa-assure-hmrc-data-production.cloud-platform.service.justice.gov.uk
+    - laa-assure-hmrc-data-production.apps.live.cloud-platform.service.justice.gov.uk
+    - assure-hmrc-data.service.justice.gov.uk
+  tls:
+    - hosts:
+      - assure-hmrc-data.service.justice.gov.uk
+      secretName: assure-hmrc-data-tls-certificate
+
+resources:
+  limits:
+    cpu: 500m
+    memory: 1G
+  requests:
+    cpu: 10m
+    memory: 125Mi
```

Comment:
we can remove this in line with similar on hrmc-interface and [recommendations from CP](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/concepts/namespace-limits.html#container-request-limits)